### PR TITLE
TEL-4691: Configurable TTS Mute/UnMute in Conference

### PR DESCRIPTION
**Pull request by dev-ryanc**

### CODE REVIEW

File: src/mod/applications/mod_conference/conference_loop.c [hunk 1]
```diff
@@ -1569,10 +1569,10 @@ void conference_loop_output(conference_member_t *member)
...
 			if (!zstr(member->conference->muted_sound)) {
 				conference_member_play_file(member, member->conference->muted_sound, 0, SWITCH_TRUE);
-			} else {
+			} else if (!zstr(member->conference->muted_tts)) {
 				char msg[512];
 
-				switch_snprintf(msg, sizeof(msg), "Muted");
+				switch_snprintf(msg, sizeof(msg), member->conference->muted_tts);
 				conference_member_say(member, msg, 0);
 			}
```
Comment (pchero) [inline src/mod/applications/mod_conference/conference_loop.c:1577]: Don't we need a default tts sound like before?
```
else {
	char msg[512];
	switch_snprintf(msg, sizeof(msg), "Muted");
	conference_member_say(member, msg, 0);
}
```

File: src/mod/applications/mod_conference/conference_loop.c [hunk 2]
```diff
@@ -1581,10 +1581,10 @@ void conference_loop_output(conference_member_t *member)
 		if (conference_utils_member_test_flag(member, MFLAG_INDICATE_MUTE_DETECT)) {
 			if (!zstr(member->conference->mute_detect_sound)) {
 				conference_member_play_file(member, member->conference->mute_detect_sound, 0, SWITCH_TRUE);
-			} else {
+			} else if (!zstr(member->conference->mute_detect_tts)) {
 				char msg[512];
 
-				switch_snprintf(msg, sizeof(msg), "Currently Muted");
+				switch_snprintf(msg, sizeof(msg), member->conference->mute_detect_tts);
 				conference_member_say(member, msg, 0);
```
Comment (pchero) [inline src/mod/applications/mod_conference/conference_loop.c:1588]: Here too
```
else {
	char msg[512];
	switch_snprintf(msg, sizeof(msg), "Currently Muted");
	conference_member_say(member, msg, 0);
}
```

File: src/mod/applications/mod_conference/conference_loop.c [hunk 3]
```diff
@@ -1593,10 +1593,10 @@ void conference_loop_output(conference_member_t *member)
 		if (conference_utils_member_test_flag(member, MFLAG_INDICATE_UNMUTE)) {
 			if (!zstr(member->conference->unmuted_sound)) {
 				conference_member_play_file(member, member->conference->unmuted_sound, 0, SWITCH_TRUE);
-			} else {
+			} else if (!zstr(member->conference->unmuted_tts)) {
 				char msg[512];
 
-				switch_snprintf(msg, sizeof(msg), "Un-Muted");
+				switch_snprintf(msg, sizeof(msg), member->conference->unmuted_tts);
 				conference_member_say(member, msg, 0);
```
Comment (pchero) [inline src/mod/applications/mod_conference/conference_loop.c:1600]: Here too
```
else {
	char msg[512];
	switch_snprintf(msg, sizeof(msg), "Un-Muted");
	conference_member_say(member, msg, 0);
}
```

**Review by pchero — COMMENTED** [review]
Don't we need a default sound like before?

File: src/mod/applications/mod_conference/conference_loop.c (hunk 1 above)
Comment (dev-ryanc) [reply src/mod/applications/mod_conference/conference_loop.c:1577]: No, they are needed to be configured in the profile instead if we want the profile to play TTS muted sound

File: src/mod/applications/mod_conference/conference_loop.c (hunk 2 above)
Comment (dev-ryanc) [reply src/mod/applications/mod_conference/conference_loop.c:1588]: No, they are needed to be configured in the profile instead if we want the profile to play TTS muted sound

File: src/mod/applications/mod_conference/conference_loop.c (hunk 3 above)
Comment (dev-ryanc) [reply src/mod/applications/mod_conference/conference_loop.c:1600]: No, they are needed to be configured in the profile instead if we want the profile to play TTS unmuted sound

**Review by pchero — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**